packages/e2e-tests/specs/widgets/customizing-widgets.test.js

```diff
@@ -763,6 +763,17 @@ describe( 'Widgets Customizer', () => {
 
 		expect( console ).toHaveErrored( twentyTwentyError );
 	} );
+
+	it( 'should not render Block Settings sections', async () => {
+		// We add Block Settings as a section, but it shouldn't display to
+		// the user as a section on the main menu. It's simply how we
+		// integrate the G sidebar inside the customizer.
+		const findAllBlockSettingsHeader = findAll( {
+			role: 'heading',
+			name: /Block Settings/,
+		} );
```

Comment:
This will wait for 3 seconds, best to provide the `timeout` option. Or just use the `$$` selector.

Reply:
I was able to move the timeout down to 0 and verify that it still fails with the fix missing, and passes with the fix present